src/pynwb/ndx_photostim/spec/ndx-photostim.namespace.yaml

```diff
@@ -0,0 +1,25 @@
+namespaces:
+- author:
+  - Carl Harris
```

Comment:
@agt24 @lafosse I think we said we'd add Paul here as co-author due to his contributions, and add him as an email contact too, yes?

I would just change this but I'm not sure if it needs to be done anywhere else.

Reply:
Yes, definitely. This yaml file was previously in a different directory, I just moved it to the spec subdirectory. Hadn't notice it contained an author field, but I've now added Paul's name and email.

Reply:
This and the other additions all look good to me. Thanks!